sampleproject/components/nested/calendar/calendar.py

```diff
@@ -0,0 +1,25 @@
+from django_components import component
+
+
+@component.register("calendar_nested")
+class CalendarNested(component.Component):
```

Comment:
Added one more variant of the calendar, CalendarNested, to confirm that the logic works even for components that are not directly under the `components` directory.

Can confirm it works!

<img width="427" alt="Screenshot 2024-03-17 at 12 04 39" src="https://github.com/EmilStenstrom/django-components/assets/25986782/3ba1b8ed-68a0-4480-9fd2-dd664ed5a2c1">